frontend/js/modules/user-invitation.js

```diff
@@ -116,11 +160,30 @@ module.exports = (function () {
         return memo;
       }, { byId: {}, byLogin: {}, byEmail: {} });
       c.users     = ld.merge(users.byLogin, users.byEmail);
-      var current = ld(users.byId)
-        .pick(c.isInvite ? c.group.users : c.group.admins)
+
```

Comment:
Refactor the behavior together again, just use conditions to determine the picklist parameter, but avoid copy pasting the ld/pick/values/pluck/value calls